src/service/authentication.ts

```diff
@@ -0,0 +1,14 @@
+import firebase from 'firebase/app'
+import { auth, provider } from './database'
+
+export const signIn = () => {
+  return auth.signInWithPopup(provider)
+}
+
+export const signOut = () => {
+  return firebase.auth().signOut()
+}
+
+export const subscriber = (observer: any) => {
```

Comment:
Could we add a similar function name to the other ones? Like subscribe, subscribeToAuthentication

Reply:
fixed